portfolio/src/main/webapp/index.html

```diff
@@ -1,18 +1,89 @@
 <!DOCTYPE html>
 <html>
-  <head>
-    <meta charset="UTF-8">
-    <title>My Portfolio</title>
-    <link rel="stylesheet" href="style.css">
-    <script src="script.js"></script>
-  </head>
-  <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
-    </div>
-  </body>
+    <head>
+        <meta charset="UTF-8">
+        <title>My Portfolio</title>
+        <link rel="stylesheet" href="style.css">
+        <script src="script.js"></script>
+    </head>
+    <body>
+        <div id="wrapper">
+            <div id="content">
+                <div id="header">
+                    <h1 id="intro">Hi. I'm Zian.</h1>
+                    <div id="external-links">
+                        <a href="mailto:zianchen774@gmail.com" target="_blank">
+                            <img class="icon" src="./images/mail_icon.png" 
+                                alt="mail_icon"/>
```

Comment:
The `alt` is used by screen readers so something like 'Mail icon' might be more suitable. I would also suggest either moving the `alt` on the same line or aligning it under so that it aligns with the start of `class`.

Reference: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/img